node/core/av-store/src/lib.rs

```diff
@@ -477,6 +483,10 @@ impl AvailabilityStoreSubsystem {
 		Self {
 			pruning_config,
 			inner,
+			// In testing we would want to test actual deletions from DB,
+			// the values in the cache will prevent that so here cache size is
+			// set to `0`.
```

Comment:
Why can you not prune properly in tests to ensure that the cache is empty as well?

Reply:
yep, you are right, this is leftover after LRU where the cache and store could diverge in their contents. Updated code to remove this